geps/gep-1494/index.md

```diff
@@ -0,0 +1,158 @@
+# GEP-1494: HTTP Auth in Gateway API
+
+* Issue: [#1494](https://github.com/kubernetes-sigs/gateway-api/issues/1494)
+* Status: Provisional
+
+(See status definitions [here](/geps/overview/#gep-states).)
+
+
+## TLDR
+
+Provide a method for configuring **Gateway API implementations** to add HTTP Authentication for north-south traffic. The method may also include Authorization config if practical. At the time of writing, this authentication is only for ingress traffic to the cluster.
+
+
+## Goals
+
+(Using the [Gateway API Personas](https://gateway-api.sigs.k8s.io/concepts/roles-and-personas/))
+
+* A way for Ana the Application Developer to configure a Gateway API implementation to perform Authentication (at least), with optional Authorization.
+
+* A way for Chihiro the Cluster Admin to configure a default Authentication and/or Authorization config for some set of HTTP or GRPC matching criteria.
+
+* Optionally, a way for Ana to have the ability to disable Authentication and/or Authorization for specific routes when needed, allowing certain routes to not be protected. This would probably need to work something like a default enabling at Gateway level, that can be specifically set at lower levels, but further design is TBD. This goal comes from the relatively-common desire for Chihiro to be able to set reasonably-secure defaults, and for Ana or others to be able to _disable_ for specific paths for purposes of healthchecking. The fact that this is relatively undefined is why this goal is _optional_.
+
+
+## Non-Goals
+
+* Handling all possible authentication and authorization schemes. Handling a (preferably large) subset of authentication and authorization is acceptable.
+
+
+## Deferred Goals
+
+* (Not performed during the Provisional Phase) Defining the API or configuration to be used.
+* Handling GRPCRoute (We will handle plain HTTP first)
```

Comment:
Slight change to avoid any suggestion on whether this should be attached at a Route or Gateway level.
```suggestion
* Handling GRPC (We will handle plain HTTP first)
```